environment.yml

```
@@ -0,0 +1,26 @@
+name: forests-offsets-fire
+channels:
+  - conda-forge
+  - nodefaults
+dependencies:
+  - python=3.10
+  - beautifulsoup4
+  - click
+  - fsspec
+  - fuzzywuzzy
```

```suggestion
  - thefuzz 
```
this package got migrated/renamed